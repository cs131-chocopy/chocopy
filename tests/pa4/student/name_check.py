def makeint()->bool:
    return True
def makebool() -> str:
    return "114514"
def makestr() -> int:
    return 5
def concat()->int:
    return -5
def print_8():
    print(8)
def alloca():
    print("alloca")
def alloc():
    print("alloc")
def streql():
    print("streql")

print(makeint())
print(makebool())
print(makestr())
print(concat())
print_8()
alloca()
alloc()
streql()
