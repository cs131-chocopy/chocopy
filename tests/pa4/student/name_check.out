True
114514
5
-5
8
alloca
alloc
streql
